docs: Rewriting pages about editors (#2552)
codenvy/che-docs: modules/end-user-guide/pages/ides-in-workspaces.adoc

:_content-type: PROCEDURE
:description: IDEs in workspaces
:keywords: configuring, workspace, editor, ide, cloud ide
:navtitle: IDEs in workspaces
:page-aliases:

[id="ides-in-workspaces"]
= IDEs in workspaces

== Supported IDEs

The default IDE in a new workspace is Microsoft Visual Studio Code - Open Source. Alternatively, you can choose another supported IDE:

.Supported IDEs 

[cols="1,1a,1a"]
|===
| IDE | `id` | Note

include::example$snip_{project-context}-table-of-supported-editors.adoc[]

|===

== Repository-level IDE configuration in {prod-short}

> You can store IDE configuration files directly in the remote Git repository that contains your project source code and that will be cloned for a new workspace. This way, one common IDE configuration will be applied to all new workspaces that will feature a clone of that repository. Such IDE configuration files might include the following:

Reviewer: ```suggestion
You can store IDE configuration files directly in the remote Git repository that contains your project source code. This way, one common IDE configuration is applied to all new workspaces that feature a clone of that repository. Such IDE configuration files might include the following:
```

Author: Thank you. I'm applying this suggestion.
